apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -178,7 +178,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
   def translate(%Candidate.Macro{name: "defguard", arity: 1} = macro, builder, env) do
     label = "#{macro.name} (Define a guard macro)"
 
-    snippet = "defguard ${1:call} $0"
+    snippet = "defguard ${1:guard}(${2:args}) when $0"
```

Comment:
Maybe not in this case, because in VS Code at least, if the snippet doesn't have a `$0` in it, it's implicitly added to the end. So I think these two are equal:

```elixir
snippet = "defguard ${1:guard}(${2:args}) when $0"
snippet = "defguard ${1:guard}(${2:args}) when "
```

I think I prefer being more explicit, though. WDYT?

Reply:
Those two are equal, 
I think this commit mentioned something about removing $0 at the end of the snippets. I don't have a preference about being explicit with tab stops.

Reply:
I'll look over them all again to make sure I didn't make any mistakes, but I think the $0 I removed were cases where it was just moving the cursor outside the expression, and that seemed weird. Like `defstruct [${1:fields}] $0` -- In those cases, I removed it because it was superfluous.

Reply:
However, I'm now thinking it should probably be `defstruct [${0:fields}]`, since not including the $0 means it's implicitly added to the end.